src/components/dashboard/FeedItems/ListEventItem.js

```diff
@@ -62,18 +62,21 @@ const InviteItem = ({ item, theme }) => (
 
 const NewsItem: React.FC = ({ item, eventSettings, styles }) => {
   const {
-    data: { sponsoredLink, sponsoredLogo },
+    data: { sponsoredLink, sponsoredLogo, picture },
   } = item
+  const { width, height } = useMemo(() => resolveAssetSource(picture), [picture])
+
   const onSponsorPress = useOnPress(() => {
     fireEvent(GOTO_SPONSOR, { link: sponsoredLink })
     Linking.openURL(sponsoredLink).catch(e => log.error('Open news feed error', e))
   }, [sponsoredLink])
+
   return (
     <View style={styles.rowContent}>
       <FeedListItemLeftBorder style={styles.rowContentBorder} color={eventSettings.color} isBig />
 
       <View style={styles.newsContent}>
-        {item.data.picture && <Image source={{ uri: item.data.picture }} style={styles.newsPicture} />}
+        {picture && <Image source={{ uri: picture }} style={[styles.newsPicture, { aspectRatio: width / height }]} />}
```

Comment:
we don't need auto crop feature. feed admins will upload images  had been prepared with some ratio.
just make width full size accoriding to design, height should be auto

Reply:
@johnsmith-gooddollar what ratio images will have? As 'height: auto' doesn't work, we need either some constant height or some ratio

Reply:
@roma-hladilka please find ways for auto height to be working

Reply:
@roma-hladilka https://reactnative.dev/docs/image#resizemode

Reply:
all libraries are depended on method, which crashes web, so ratio is the only way I see

Reply:
resize mode = contain **should** work.

Reply:
please google about 'react native image auto height'